ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/RestIntegrationTestUtil.java

```diff
@@ -71,6 +72,19 @@
   private RestIntegrationTestUtil() {
   }
 
+  public static HealthCheckResponse makeHealthCheck(final TestKsqlRestApp restApp) {
```

Comment:
For one moment, I got confused with the name thinking you were creating a health check object.

```suggestion
  public static HealthCheckResponse checkServerHealth(final TestKsqlRestApp restApp) {
```

Reply:
done